tests/handlers/test_users.py

```diff
@@ -368,7 +371,7 @@ def test_changing_email_ends_any_other_sessions_of_current_user(self):
 
             # force the old `user_id`, simulating that the user is logged in from another browser
             with c.session_transaction() as sess:
-                sess["user_id"] = previous
+                sess["_user_id"] = previous
             rv = self.get_request("/api/users/{}".format(self.factory.user.id))
```

Comment:
> We should probably try to fix it, as it's testing something important (session/identity behaviour). How is the unit test failing?

The problem lies in this test. By simulating login with the old user_id, I took a quick look and it seems that in the new version of flask_login, the new user_id can be obtained through a cookie, which allows for successful login.